analytics_data_api/v0/models.py

```diff
@@ -112,6 +112,25 @@ class Meta(object):
     created = models.DateTimeField(auto_now_add=True)
 
 
+class ProblemResponseAnswerDistribution(BaseProblemResponseAnswerDistribution):
+    """ Original model for the count of a particular answer to a response to a problem in a course. """
+
+    class Meta(BaseProblemResponseAnswerDistribution.Meta):
+        managed = False
+
+    count = models.IntegerField()
+
+
+class ProblemFirstLastResponseAnswerDistribution(BaseProblemResponseAnswerDistribution):
```

Comment:
What do you think about renaming this to `ProblemFirstFinalResponseAnswerDistribution`?  It seems like "final" is used rather than "last" in the member variables.